components/system-broker/internal/osb/bind_last_op.go

```diff
@@ -19,16 +19,90 @@ package osb
 import (
 	"context"
 
+	schema "github.com/kyma-incubator/compass/components/director/pkg/graphql"
+	"github.com/kyma-incubator/compass/components/system-broker/internal/director"
 	"github.com/kyma-incubator/compass/components/system-broker/pkg/log"
 	"github.com/pivotal-cf/brokerapi/v7/domain"
+	"github.com/pivotal-cf/brokerapi/v7/domain/apiresponses"
 	"github.com/pkg/errors"
 )
 
 type BindLastOperationEndpoint struct {
+	credentialsGetter packageCredentialsFetcher
 }
 
 func (b *BindLastOperationEndpoint) LastBindingOperation(ctx context.Context, instanceID, bindingID string, details domain.PollDetails) (domain.LastOperation, error) {
 	log.C(ctx).Infof("LastBindingOperation instanceID: %s bindingID: %s details: %+v", instanceID, bindingID, details)
 
-	return domain.LastOperation{}, errors.New("not supported")
+	opType := details.OperationData
+	appID := details.ServiceID  // may be empty per OSB spec
+	packageID := details.PlanID // may be empty per OSB spec
+	authID := bindingID
+
+	logger := log.C(ctx).WithFields(map[string]interface{}{
+		"opType":     opType,
+		"appID":      appID,
+		"packageID":  packageID,
+		"authID":     authID,
+		"instanceID": instanceID,
+		"bindingID":  bindingID,
+	})
+
+	logger.Info("Fetching package instance credentials")
+	resp, err := b.credentialsGetter.FetchPackageInstanceAuth(ctx, &director.PackageInstanceInput{
+		InstanceAuthID: authID,
+		Context: map[string]string{
+			"instance_id": instanceID,
+			"binding_id":  bindingID,
+		},
+	})
+	if err != nil && !IsNotFoundError(err) {
+		return domain.LastOperation{}, errors.Wrapf(err, "while getting package instance credentials from director")
+	}
+
+	if IsNotFoundError(err) {
+		logger.Info("Package instance credentials not found")
```

Comment:
This is something expected to happen if the op is Unbind, but is abnormal if the operation is Bind. May be put the log on Error level is move it after the If.